dashboard/server/api/runApi.js

```diff
@@ -9,47 +9,66 @@ const {
     FRONTEND_RUN_STATS,
 } = require('../../lib/constants/frontend-events');
 
+const { asyncWrapper } = require('../../lib/utils');
+const controllers = require('../../lib/controllers/run');
+
 module.exports = (socket) => ({
-    handleNewRun: (data, callback) => {
+    handleNewRun: asyncWrapper(async (data, callback) => {
```

Comment:
Need to get rid of `asyncWrapper`.

Reply:
Do you want to embed them in `try-catch` blocks?